life/modernized/06-chapter-5.adoc

```diff
@@ -181,18 +192,30 @@ that his dread may go along with us, over all.
 As you can, at opportunity, write to us, it will make me glad;
 the face of one Friend would rejoice my soul.
 
+--
+
+[.offset]
 In a postscript he adds,
 
+[.embedded-content-document.letter]
```

Comment:
I had at first used the [.postscript] thing for this, but I thought it looked better this way. But maybe you want it as a postscript, just let me know.